webpack.config.client.js

```diff
@@ -72,6 +79,7 @@ module.exports = ({
         : prodPublicPath,
     },
     optimization: {
+      moduleIds: 'deterministic',
```

Comment:
From [the update-outdated-options section](https://webpack.js.org/migrate/5/#update-outdated-options) of the webpack upgrade docs 

> HashedModuleIdsPlugin → optimization.moduleIds: 'deterministic'